### ISSUE TITLE
Steam games runs on black screen with sound after system unlock

### ISSUE THREAD

**Issue by eth481642** [ValveSoftware/steam-for-linux#7866]
#### Your system information

* Steam client version: Built Jun 08 2021 (latest)
* Distribution: Ubuntu 20.04.2 (Gnome 3, proprietary Nvidia drivers) 
* Opted into Steam client beta?: No
* Have you checked for system updates?: Yes

#### Please describe your issue in as much detail as possible:
Before writing this issue I googled a lot, and I found many other people with same issue, but no one found the reason and solution for this. I also found that this issue exists for a long time (>1year)

When I lock my graphical session by pressing Win+L or Ctrl+Alt+L and then unlock it after some time, steam games launches with black screen only but with sound. When I try to switch between apps via Alt+Tab, I can see the game window and the game itself in it, but when I release Alt+Tab on game window, screen goes black again. I found 3 solutions for this:
1. Reboot
2. Log out graphical session and log back again
3. Run specific native linux game. This one is really strange. I managed to found at least 2 linux games that fixes the issue: Left 4 Dead 2 and Metro 2033 Redux. When the problem exists (after screen unlock), if I run this games, they run normally (without black screen), then I exit the game and after that I can launch every game without black screen (until next screen lock). If I lock my screen and then unlock it after some time, I need to do this procedure again to fix all games

I tried to run steam via terminal to watch log output and compare the problem instance(when games runs with black screen) and normal instance. I compared 2 log files with diff and there are no differencies except lines with dates and PIDs

#### Steps for reproducing this issue:

1. Lock your graphical session by pressing Win+L or Ctrl+Alt+L
2. Let your monitor to turn off and wait for 10 minutes
3. Unlock your computer and launch some linux games

**Comment by kisak-valve** [comment]
Hello @eth481642, this reads more like a desktop environment or video driver issue than an issue with Steam. It may be worthwhile to test other desktop environments and also mention the issue to your video driver vendor if it hasn't been already.

**Comment by jimbojd72** [comment]
I had the exact problem has you on arch. Reproducing the black screen after just turning off/on the monitor. Enabled ForceCompositionPipeline but now the game is stuck on the initial loading bar :cry:

**Comment by eth481642** [comment]
> I had the exact problem has you on arch. Reproducing the black screen after just turning off/on the monitor. Enabled ForceCompositionPipeline but now the game is stuck on the initial loading bar

When I selled my monitor with G-SYNC and moved to monitor without G-SYNC the problem went away. So, just want to make it clear - do you have G-SYNC enabled monitor?

**Comment by jimbojd72** [comment]
> > I had the exact problem has you on arch. Reproducing the black screen after just turning off/on the monitor. Enabled ForceCompositionPipeline but now the game is stuck on the initial loading bar
> 
> When I selled my monitor with G-SYNC and moved to monitor without G-SYNC the problem went away. So, just want to make it clear - do you have G-SYNC enabled monitor?

Yes, it is gsync enabled. I may try to disable it at the monitor level 🤷 

I tried to disabled gsync in nvidia-settings without success though.

**Comment by jimbojd72** [comment]
> > I had the exact problem has you on arch. Reproducing the black screen after just turning off/on the monitor. Enabled ForceCompositionPipeline but now the game is stuck on the initial loading bar
> 
> When I selled my monitor with G-SYNC and moved to monitor without G-SYNC the problem went away. So, just want to make it clear - do you have G-SYNC enabled monitor?

You were right. Disabling gsync directly on the monitor instead of in the nvidia-settings did the trick :1st_place_medal: 

I cannot thank you enough @eth481642 !

**Comment by Jarjarfan** [comment]
SOLVED!
I was going round and round with different solutions, tweaking proton compatibility layers, switching sound output, and looking for Nvidia setting solutions .  None of them worked. 
I'm running on CachyOS, which is based on Arch but I'm guessing solution will be simliar for others.
I went into the system's general display settings and changed the option for  
Legacy applications (X11) to 
Scaled by the system
instead of
Apply Scaling themselves
